Microsoft.Toolkit.Uwp.UI.Controls/RadialGauge/RadialGauge.cs

```diff
@@ -191,9 +200,17 @@ public RadialGauge()
         {
             DefaultStyleKey = typeof(RadialGauge);
 
+            ThemeListener.ThemeChanged -= ThemeListener_ThemeChanged;
+            ThemeListener.ThemeChanged += ThemeListener_ThemeChanged;
```

Comment:
Can't this leak? Shouldn't we do that on the OnApplyTemplate method? Btw, the PointerReleased over there is not being unassigned.

Reply:
Agree on moving the registration to OnApplyTemplate (since the handler forces a redraw). Where would be a place to unassign it?